### PR TITLE
feat: modal

### PR DESCRIPTION
**Pull request by irajbhinde**
Hey! Please review my code & let me know if any changes could be made.

Added Component - 

Modal

GIFY - 
modal

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (maitrakhatri) [inline .vscode/settings.json:3]: add this in git ignore file

File: Components/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  color: var(--white-text-color);
+}
+
+.modal-bg {
+  position: fixed;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 100vh;
+  background-color: rgb(0, 0, 0, 0.5);
```
Comment (maitrakhatri) [inline Components/modal/modal.css]: use variables for color

File: Components/modal/modal.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.bg-active {
+  visibility: visible;
+  opacity: 1;
+}
+
+.close-modal {
+  color: var(--white-text-color);
+  font-size: 28px;
```
Comment (maitrakhatri) [inline Components/modal/modal.css]: you have used rem earlier for spacing, so here also use rem for consistency

File: Components/modal/modal.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  font-weight: bold;
+  top: 5px;
+}
+
+.close-modal:hover,
+.close-modal:focus {
+  color: black;
```
Comment (maitrakhatri) [inline Components/modal/modal.css]: use variables or hex code for colors, as different browser interprets color differently.

**Review by maitrakhatri — APPROVED** [review]
Have left some suggestions, rest looks good.

**Review by akash-1318 — COMMENTED** [review]
All the points raised by Maitra are good apart from that no such changes require from my end.

**Pull request merged**